Recon/python/simpleTrigger.py

```diff
@@ -40,7 +40,10 @@ def __init__(self, name, beamEnergy) :
         super().__init__(name,'recon::TriggerProcessor','Recon')
           
         self.beamEnergy = beamEnergy
-        self.thresholds = [ self.beamEnergy/4000.*1500.0, self.beamEnergy/4000.*1000. + self.beamEnergy, self.beamEnergy/4000.*500. + 2*self.beamEnergy, self.beamEnergy/4000.*100. + 3*self.beamEnergy ]
+        if (self.beamEnergy == 4000.): 
+            self.thresholds = [ 3000, 10000, 17000, 24200 ]
```

Comment:
```suggestion
            self.thresholds = [ 3000., 10000., 17000., 24200. ]
```
sorry I think these have to be floats